app/controllers/omninav_controller.rb

```diff
@@ -0,0 +1,7 @@
+class OmninavController < ApplicationController
+  def demo
+    builder = ::Omninav::Builder.new
+    omninav_html = builder.build_html
+    render html: omninav_html, layout: 'omninav'
+  end
+end
```

Comment:
This controller provide a page on dev server where OmniNav can be tested:

http://localhost:3000/omninav